src/Components/Navbar.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
+import './Navbar.css';
+import Person from './download.png';
+
+function Navbar() {
+  return (
+    <div className="Navbar">
+      <ul>
+        <li>
+          Bookstore CMS
+        </li>
+        <li>
+          <a href="./Books">Books</a>
+        </li>
+        <li>
+          <a href="./Categories">Categories</a>
+        </li>
+      </ul>
+      <img src="Person" alt="Person" />
+    </div>
+  );
+}
```

Comment:
- [ ] We are trying to build Single Page Applications here. That means the browser should not reload. Kindly use `<Link>`s from the `react-router-dom` package for your Navbar links, instead of the `<a>` tags. Ensure there is no browser reload in your app.